config/samples/everest_v1alpha1_monitoringconfig.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: everest.percona.com/v1alpha1
+kind: MonitoringConfig
+metadata:
+  labels:
+    app.kubernetes.io/name: monitoringconfig
+    app.kubernetes.io/instance: monitoringconfig-sample
+    app.kubernetes.io/part-of: everest-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: everest-operator
+  name: monitoringconfig-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
I know I'm also guilty of completely ignoring this in my previous PRs but we need to start filling in these samples.